test/package/CMakeLists.txt

```diff
@@ -30,9 +30,9 @@ if(HIP_PLATFORM STREQUAL "nvcc")
 endif()
 
 # Find rocRAND
-find_package(rocrand REQUIRED CONFIG HINTS ${rocrand_DIR} PATHS "/opt/rocm/rocrand")
+find_package(rocrand REQUIRED CONFIG HINTS ${rocrand_DIR} PATHS "${CMAKE_INSTALL_PREFIX}/rocrand")
 # Find hipRAND
-find_package(hiprand REQUIRED CONFIG HINTS ${hiprand_DIR} PATHS "/opt/rocm/hiprand")
+find_package(hiprand REQUIRED CONFIG HINTS ${hiprand_DIR} PATHS "${CMAKE_INSTALL_PREFIX}/hiprand")
```

Comment:
Not sure what "changes to support reloctable ROCM installation" means. An explanation of what that means would be helpful to evaluate these changes.
Can CMAKE_INSTALL_PREFIX could be set to "/opt/rocm" where /opt/rocm is a symlink? 
If prefix is set to /opt/rocm will be package overwrite the symlink? Probably should test those scenarios, and explain how "relocatable" installation is supported in documentation.

Reply:
Its actually multiple ROCM installation, I have updated the comment.
When CMAKE_INSTALL_PREFIX is set /opt/rocm and it is symlink.  The symlink is not overwritten, the packages gets installed into the folder it is pointing too.

Reply:
Would it be ok to overwrite the contents of /opt/rocm symlink? Could that potentially overwrite the incorrect rocm release with the package contents, say, when a built package is installed weeks later by when /opt/rocm symlink could have been modified? Could make it difficult to track install errors I think.